reviewrot/pagurestack.py

```diff
@@ -75,11 +73,15 @@ def request_reviews(self, user_name, repo_name=None, state_=None,
             try:
                 repo_reference = os.path.join(res['project']['namespace'],
                                               res['project']['name'])
-            except:
+            except AttributeError:
```

Comment:
Shoulnd't this rather be `KeyError`?

Reply:
It fails with AttributeError because res['project']['namespace'] returns None, so os.path.join is trying to join None with string. I will change it anyway to make it more clear.